TAO/TAO_IDL/be/be_predefined_type.cpp

```diff
@@ -222,9 +225,17 @@ be_predefined_type::compute_tc_name ()
                  Identifier (tcname));
         break;
       }
-    default:
-      ACE_ERROR ((LM_WARNING, "Unknown or invalid predefined type"));
+    case AST_PredefinedType::PT_uint8:
+      ACE_NEW (id, Identifier ("_tc_Uint8"));
```

Comment:
`_tc_uint8`, all others don't have a capital U

Reply:
I don't really know much about typecode and the goal here just is basic support, so that depends on if that's trivial or not.

Reply:
Wouldn't it be better to avoid TypeCode completely until there is a spec for it?